StreamChatSwiftUI.xcodeproj/project.pbxproj

```diff
@@ -2732,20 +2743,12 @@
 				version = 4.10.1;
 			};
 		};
-		848399E6275FB3BE003075E4 /* XCRemoteSwiftPackageReference "swift-snapshot-testing" */ = {
-			isa = XCRemoteSwiftPackageReference;
-			repositoryURL = "https://github.com/pointfreeco/swift-snapshot-testing.git";
-			requirement = {
-				kind = exactVersion;
-				version = 1.11.0;
-			};
-		};
 		84E95A75284A486600699FD3 /* XCRemoteSwiftPackageReference "stream-chat-swift" */ = {
 			isa = XCRemoteSwiftPackageReference;
 			repositoryURL = "https://github.com/GetStream/stream-chat-swift.git";
 			requirement = {
-				kind = upToNextMajorVersion;
-				minimumVersion = 4.39.0;
+				branch = ci/spm;
+				kind = branch;
```

Comment:
This change should be reverted after review. The PR itself should be only merged straight before the next release.
cc: @martinmitrevski

Reply:
Yes, we shouldn't merge this, but wait for the release of StreamChat.

Reply:
Yeah, so I'm returning `4.39.0` back for now. All checks will fail, but it's expected. After releasing `4.40.0` I will merge `main` into my branch and wait for the checks to pass.